### DISCUSSION TITLE
Expose tags in schedule API

### DISCUSSION THREAD

**Discussion by ligi** [pretalx/pretalx#1536] [General]
currently tags are exposed via:
https://docs.pretalx.org/api/resources/tags.html

but we find no way to get the tags for a session. We will build our schedule display on our own and the tags would make it nicer. So far I would not know how the existing endpoint could be useful without having a way to get the tags for a session. Maybe this is a low hanging fruit that is way less complex than using it in the default pretalx schedule widget? We already use the current main branch - so we are not bound to a release. Thanks for considering.

**Comment by q9f** [comment]
The unser interface informs the organizer that tags will be visible in future releases. Is there any tracking issue where we can follow the progress?

**Comment by rixx** [comment]
Please note that the schedule API is not documented and very much unofficial, and even more likely to change than the entire API (in the upcoming rewrite).

You can currently export tags using the export functionality. Any API changes are blocked on the currently pending API rewrite.
